packages/vite/src/node/__tests__/plugins/json.spec.ts

```diff
@@ -35,6 +35,22 @@ describe('transform', () => {
     return (plugin.transform! as Function)(input, 'test.json').code
   }
 
+  test("namedExports: true, stringify: 'auto' should not transformed an array input", () => {
+    const actualSmall = transform(
+      '[{"a":1,"b":2}]',
+      { namedExports: true, stringify: 'auto' },
+      false,
+    )
+    expect(actualSmall).toMatchInlineSnapshot(`
+"export default [
+	{
+		a: 1,
+		b: 2
+	}
+];"
+    `)
+  })
+
```

Comment:
Would you add a test for the following input?
```js
transform(
  '[{"a":1,"b":2}]',
  { namedExports: true, stringify: true },
  false,
)
```

Reply:
done